gerrit.go

```diff
@@ -265,18 +271,22 @@ func (c *Client) DeleteRequest(urlStr string, body interface{}) (*Response, erro
 	return c.Do(req, nil)
 }
 
-// RemoveMagicPrefixLine removed the "magic prefix line" of Gerris JSON response.
-// the JSON response body starts with a magic prefix line that must be stripped before feeding the rest of the response body to a JSON parser.
-// The reason for this is to prevent against Cross Site Script Inclusion (XSSI) attacks.
+// RemoveMagicPrefixLine removes the "magic prefix line" of Gerris JSON
+// response if present. The JSON response body starts with a magic prefix line
+// that must be stripped before feeding the rest of the response body to a JSON
+// parser. The reason for this is to prevent against Cross Site Script
+// Inclusion (XSSI) attacks.  By default all standard Gerrit APIs include this
+// prefix line though some plugins may not.
 //
 // Gerrit API docs: https://gerrit-review.googlesource.com/Documentation/rest-api.html#output
 func RemoveMagicPrefixLine(body []byte) []byte {
-	index := bytes.IndexByte(body, '\n')
-	if index > -1 {
-		// +1 to catch the \n as well
-		body = body[(index + 1):]
+	if bytes.HasPrefix(body, []byte(")]}'\n")) {
+		index := bytes.IndexByte(body, '\n')
+		if index > -1 {
+			// +1 to catch the \n as well
+			body = body[(index + 1):]
+		}
 	}
```

Comment:
@shurcooL, no issues here if you want to go ahead and make the changes. I'd do it myself but I don't currently have my local host setup to make the change and test it.